docs/getting_access/native_installation.md

```diff
@@ -69,7 +69,49 @@ The good news is that all of this only requires a handful commands :astonished:
     This is good enough for an individual client, or for testing purposes,
```

Comment:
I would:
- Remove this whole note. 
- Create a subtile right below "Native installation" with `## Installation for single clients`
- Then, at the end of the _current_ docs, make a new subtitle `## Installation for HPC (or otherwise large) systems`. Under that header, first explain that such systems are adviced to use (multiple) Squid(s) for load balancing (~1 per 500 nodes) and a full Stratum 1 for redundancy. Say that the docs here will explain how to configure the client for that, and refer to the other pages (e.g. ../filesystem_layer/stratum1.md) to learn how to setup a Stratum 1 or Squid.

Reply:
Done in https://github.com/EESSI/docs/pull/157/commits/9e97d4fc36dc185e75ef9dc0f90f7460cb536f20.